src/test/java/hudson/plugins/sonar/CasCTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * SonarQube Scanner for Jenkins
+ * Copyright (C) 2007-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package hudson.plugins.sonar;
+
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import jenkins.model.GlobalConfiguration;
+import org.junit.Test;
+
+import java.net.URL;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CasCTest extends SonarTestCase {
+
+    @Test
+    public void testMissingServerAuthenticationToken() throws Exception {
```

Comment:
See https://github.com/jenkinsci/antisamy-markup-formatter-plugin/pull/13 for a better test pattern